apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -261,16 +275,22 @@ func (mm *MachineManagerAPI) DestroyMachineWithParams(args params.DestroyMachine
 	for i, tag := range args.MachineTags {
 		entities.Entities[i].Tag = tag
 	}
-	return mm.destroyMachine(entities, args.Force, args.Keep)
+	return mm.destroyMachine(entities, args.Force, args.Keep, args.MaxWait)
 }
 
-func (mm *MachineManagerAPI) destroyMachine(args params.Entities, force, keep bool) (params.DestroyMachineResults, error) {
+func (mm *MachineManagerAPI) destroyMachine(args params.Entities, force, keep bool, maxWait *time.Duration) (params.DestroyMachineResults, error) {
 	if err := mm.checkCanWrite(); err != nil {
 		return params.DestroyMachineResults{}, err
 	}
 	if err := mm.check.RemoveAllowed(); err != nil {
 		return params.DestroyMachineResults{}, err
 	}
+
+	if maxWait == nil {
+		defaultMaxWait, _ := time.ParseDuration("0s")
```

Comment:
check with xtian about defauls here

Reply:
I think the default should be 1 minute here rather than 0 - if `--no-wait` isn't specified that's what we'd want.